### ISSUE TITLE
Person is created but are not shown in the People list.

### ISSUE THREAD

**Issue by hitech95** [immich-app/immich#13551]
### The bug

Issue #13245 is part of a deeper problem on the API for people.
While PR #13288 try to mitigate it it open a door to other issues.

When a face detection meet threshold person is created;
 but people list doesnt show the newly created person, while
 the counter is updated.

The photo that triggered the person creation has the person is
 correctly assigned to the person but the other "faces" that contributed
 to the threshold are not added.
 
The same happends when a photo has a face and you manually create
 a person for such picture. But is _seems_ that by assigning a name to the
 person it trigger some logic and the person is shown.

This also means that I cannot assign the new created person to other
 image via the sidecard as the API still return only the person with at
 least `facialRecognition.minFaces`.

I think that the underlying issues are 2:
 - A person need to match the ML face threshold to be shown.
 - When person is made (via face detection or manually from the photo
    sidecard) the job to match the faces is not queued.
 
This create the mismatch between the counter and the list.
As the list require the threshold to be matched to show an element.
 
I'm seeing the following solutions:
1. Trust that a person is generated only if the threshold is meet or it has been done my an user.
  If a person has been generated and has at lease one picture then is always show.
  AFIK the person is always shown if has a name even if it doesnt meet the `facialRecognition.minFaces` threshold but
  it could be a side effect of assigning a name after the personc reation that trigger some other background job)
2. Queue a face match scan on the full library each time a new person has been created.
   This automatically re-scan the library to connect the faces to the right person.
   This is resource intensive for nothing
3. Attach the feces (matches) used to calculate the threshold to such person.
4. The sidecard to attach a person to an unmatched face also includes the persons that haven't met the threshold.

I think that the best scenario would be to implement 1,2,4.

### The OS that Immich Server is running on

Debian 12

### Version of Immich Server

1.118.2

### Version of Immich Mobile App

ND

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
NA
```

### Your .env content

```Shell
NA
```

### Reproduction steps

1. set `facialRecognition.minFaces` to 3 (default)
2. upload 3 picture of the same person
3. the last uploaded picture trigger the person creation
4. the other 2 pictures still have the unmatched person

**Comment by mlinko6** [comment]
I have a similar Issue, I create new people but are not shown in the People - and can't be found either... There is also a thing that if I "hide" people not all faces that are still not hidenn are shown... I can scroll in the hidden section to hide some more people and save the changes, but when I open the people and hidden faces again there are the same faces still unhidden allthough I've just hidden them...

**Comment by jpaffrath** [comment]
I dont know if I encounter the same bug, but I see unassigned faces in my images info panel, which are not visible under my peoples list. There are not hidden either. However, in the info panel, the unassigned faces are hidden and I cannot create new people from there, just assign them to already existing people.

<img width="175" alt="Image" src="https://github.com/user-attachments/assets/807c2d9d-1db3-4ebd-9476-0030b9321b48" />

**Comment by Flunkyball** [comment]
After upgrading to 1.136.0 I find myself with the same issue.

According the the server log, unidentified people are detected on new images. However they do not show up in the people list and therefore cannot be assigned.

**Comment by jrasm91** [comment]
Closing in favor of #21279

**Comment by hitech95** [comment]
> Closing in favor of [#21279](https://github.com/immich-app/immich/issues/21279)

I dont see how the new threshold would "fix" the issue.
This is not a direct issue of the minimum threashold as I can manually create a person, but it is not extracted to assign it to the detected ML faces.

On that ticket there is no mention about this specific issue.

**Comment by jrasm91** [comment]
Your statement does not make sense. Please be more specific and precise as you describe what exactly you are expecting to happen and what is not happening, otherwise I cannot help you.

**Comment by hitech95** [comment]
> Your statement does not make sense. Please be more specific and precise as you describe what exactly you are expecting to happen and what is not happening, otherwise I cannot help you.

Just read the ticket message. It is in there.

TLDR: we can have a situation where a Person "entity" in the DB is present (creted by a user using the sidebar or whatever) but such person even if present due to the varius threshold met is still not shown/available to assign a face.

Having a per account threshold dosn't change the underlying issue unless in the other ticket is specified a proper intender behaviour in those edge cases.

**Comment by jrasm91** [comment]
> is still not shown/available to assign a face.

The list of people that can be selected are from the `/people` endpoint, which only shows people with at least x faces.

**Comment by hitech95** [comment]
> The list of people that can be selected are from the `/people` endpoint, which only shows people with at least x faces.

The issue is that unless the job runs and assign the faces to the person such person is not shown.
you can have 3 photos with the same person and a threshold of 3.

The person is created, but assinged only on the last picture until the job runs.
As the endpoint is the same you cant choose a person to manually tag the missing pictures too.

**Comment by jrasm91** [comment]
This is working intended/designed. There is a nightly job that takes care of assigning the other 2 faces to the person.

**Comment by hitech95** [comment]
> This is working intended/designed. There is a nightly job that takes care of assigning the other 2 faces to the person.

Yea but it still is a bad UX, so unless you mark this as wontfix please re-open it.
